docs/src/guides/wordpress/composer/migrate.md

```diff
@@ -137,36 +143,54 @@ Like WordPress core itself, themes and plugins can be installed as dependencies
     }
     ```
 
-    Now any plugin that contains as a part of its package metadata a `type` of `wordpress-plugin` is installed into `wordpress/wp-content/plugins/` instead of `vendor`, and the same goes for themes and must-use plugins. 
+    After inspecting package metadata, Composer now installs the plugins whose `type` matches `wordpress-plugin` into `wordpress/wp-content/plugins/` instead of `vendor`, and the same goes for themes and must-use plugins. 
 
```

Comment:
:lady_beetle: If the directory is changed, then the `.gitignore` also has to be changed. Since the previous flow recommended a very specific `.gitignore` file, that seems like something that should be mentioned here.